packages/manager-components/src/components/content/price/price.component.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+import {
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+  ODS_TEXT_COLOR_HUE,
+} from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import './translations/translations';
+import {
+  OvhSubsidiary,
+  ovhSubsidiaryToCurrency,
+  CentsUnit,
+  Frequencytype,
+} from '../../../enumTypes';
+
+export interface PriceProps {
+  price: number;
+  frequency?: Frequencytype;
+  unit?: CentsUnit;
+  isHT?: boolean;
+  isTTC?: boolean;
+  includeGST?: boolean;
+  excludeGST?: boolean;
+  ovhSubsidiary?: OvhSubsidiary;
+  taxRate?: number;
```

Comment:
I think `taxRate` could be removed as prices always come from the API with the right values (`withTax`, `withoutTax`, ...). Also, I can't find a place where we do conversion in the manager code